pkg/api/dashboard_permission_test.go

```diff
@@ -166,6 +166,33 @@ func TestDashboardPermissionAPIEndpoint(t *testing.T) {
 				})
 		})
 
+		t.Run("When trying to update team or user permissions with a role", func(t *testing.T) {
+			role := models.ROLE_EDITOR
+			cmds := []dtos.UpdateDashboardAclCommand{
+				{
+					Items: []dtos.DashboardAclUpdateItem{
+						{UserId: 1000, Permission: models.PERMISSION_ADMIN, Role: &role},
+					},
+				},
+				{
+					Items: []dtos.DashboardAclUpdateItem{
+						{TeamId: 1000, Permission: models.PERMISSION_ADMIN, Role: &role},
+					},
+				},
+			}
+
+			for _, cmd := range cmds {
+				updateDashboardPermissionScenario(t, "When calling POST on", "/api/dashboards/id/1/permissions",
+					"/api/dashboards/id/:id/permissions", cmd, func(sc *scenarioContext) {
+						callUpdateDashboardPermissions(sc)
+						assert.Equal(t, 400, sc.resp.Code)
+						respJSON, err := simplejson.NewJson(sc.resp.Body.Bytes())
```

Comment:
Can we use the standard json package instead of simplejson? I think we want to move away from the latter.